src/main/java/ca/ciccc/Assignment1.java

```diff
@@ -38,7 +44,20 @@ public static String inchesToMeters(int inches) {
      */
     public static String addDigits(int number) {
 
-        return "";
+        int firstDigit = number % 10;
+        int remainingNumber = number / 10;
+        int SecondDigit = remainingNumber % 10;
+        remainingNumber = remainingNumber / 10;
+        int thirdDigit = remainingNumber % 10;
+        remainingNumber = remainingNumber / 10;
+        int fourthDigit = remainingNumber % 10;
+        int sum = firstDigit + thirdDigit + SecondDigit + fourthDigit;
+
```

Comment:
You can try to change your code so that it works for any number of digits (using loops)
This code only works when you have 3 digits number.